shell/platform/windows/flutter_window.cc

```diff
@@ -277,4 +281,23 @@ PointerLocation FlutterWindow::GetPrimaryPointerLocation() {
   return {(size_t)point.x, (size_t)point.y};
 }
 
+void FlutterWindow::OnThemeChange() {
+  HIGHCONTRAST high_contrast = {sizeof(HIGHCONTRAST)};
+  if (SystemParametersInfoW(SPI_GETHIGHCONTRAST, sizeof(HIGHCONTRAST),
+                            &high_contrast, 0)) {
+    BOOL hc_on = high_contrast.dwFlags & HCF_HIGHCONTRASTON;
+    // Currently, only FlutterWindowsView should be used as delegate on Windows
+    FlutterWindowsView* view =
+        reinterpret_cast<FlutterWindowsView*>(binding_handler_delegate_);
+    FlutterWindowsEngine* engine = view->GetEngine();
+    engine->UpdateHighContrastEnabled(hc_on);
+  } else {
+    FML_LOG(ERROR) << "Failed to get status of high contrast feature";
```

Comment:
I believe this API is expected to fail on Win7. Because of that I'd just ignore this failure. You could add a comment above the conditional that this is only supported on Windows 8+.

Reply:
Do you think it would still be beneficial to log something, maybe just to the `INFO` level?

Reply:
Yep info is fine! Thanks!